### PR TITLE
Add `aliases` support for `add-icon-data` script

### PR DESCRIPTION
**Pull request by LitoMore**
Related to #7849 

The `aliases` field is more complicated. We may need a bunch of questions to collect users' inputs.

Here is the type overview:

```ts
type Aliases = {
	aka?: string[];
	dup?: SimpleIcon[];
	loc?: Record<langageCode, string>;
	old?: string[];
}
```

Maybe we could add a tip to contributing doc to let users modify the `_icons/simple-icons.json` directly.

### CODE REVIEW

**Comment by LitoMore** [comment]
@mondeja Any thoughts about this? If we need to add aliases support, we will need to add a lot of prompts.

File: scripts/utils.js [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import path from 'node:path';
 import fs from 'node:fs/promises';
 import { fileURLToPath } from 'node:url';
+import { readFile } from 'node:fs';
```
Comment (mondeja) [inline scripts/utils.js:9]: This import is not used 
```suggestion
```

File: scripts/add-icon-data.js [hunk 2]
```diff
@@ -127,11 +178,12 @@ const dataPrompt = [
 const answers = await inquirer.prompt(dataPrompt);
 const icon = getIconDataFromAnswers(answers);
 
-if (answers.confirm) {
+if (answers.confirmToAdd) {
   iconsData.icons.push(icon);
   iconsData.icons.sort((a, b) => collator.compare(a.title, b.title));
   await writeIconsData(iconsData);
 } else {
+  console.log(answers);
```
Comment (mondeja) [inline scripts/add-icon-data.js:186]: This should be removed
```suggestion
```

**Review by mondeja — CHANGES_REQUESTED** [review]

**Comment by mondeja** [comment]
It works nice 🌟  Seems like adding the `loc` field is not too much useful for the script, maybe in the future.

Feel free to merge it after resolve the suggestions 🔧

**Pull request merged**